Allow Swagger UI CSS and JS paths to be configurable (#2908)
domaindrivendev/Swashbuckle.AspNetCore: README.md

}
```

### Change CSS or JS Paths ###

> By default, the Swagger UI inclide default CSS and JS, but if you need to change path or URL (ex CDN):

Reviewer: ```suggestion
By default, the Swagger UI include default CSS and JS, but if you wish to change the path or URL (for example to use a CDN):
```
